src/irc/core/irc-servers.c

```diff
@@ -201,6 +202,10 @@ static void server_init(IRC_SERVER_REC *server)
 
 	conn = server->connrec;
 
+	// server->cap_queue = g_slist_append(server->cap_active, "identify-msg");
+	server->cap_queue = g_slist_append(server->cap_active, "sasl");
```

Comment:
The latest iteration provides a better cap_queue that should be nice enough to expose to scripts